public/js/user.js

```diff
@@ -0,0 +1,86 @@
+document.getElementById("edit_location").addEventListener("click", getCurrentLocationData);
+document.getElementById("edit_location_form").addEventListener("submit", handleEditLocationSubmit);
+document.getElementById("edit_about").addEventListener("click", getCurrentAboutData);
+document.getElementById("edit_about_form").addEventListener("submit", handleEditAboutSubmit);
+
+async function handleEditLocationSubmit(event) {
+    event.preventDefault();
+
+    let city = document.getElementById("city").value;
+    let state = document.getElementById("state").value;
+
+    let data = {
+      city: city,
+      state:  state,
+      // email:      email,
+      // key:        pem
+    };
+    let payload = data;
+    // let payload = await encryptUsingRSA(SERVER_KEY, data);
+
+
+    // Encrypt payload with server public key
+    // POST data from form
+    postDataToUrl("/api/user/update", payload);
+    window.location.pathname = "/user/5432";
```

Comment:
Where do we want to redirect the user to? Not sure if we want to direct the user to that same user every time

Reply:
I think back to the logged in user's profile.  I can make it dynamically change.